core/cloudflow-cr-generator/src/main/scala/cloudflow/cr/Generator.scala

```diff
@@ -0,0 +1,154 @@
+/*
+ * Copyright (C) 2021 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package cloudflow.cr
+
+import com.typesafe.config.{ ConfigFactory, ConfigRenderOptions }
+
+import java.io.File
+import java.net.URL
+import scala.collection.JavaConverters._
+import scala.io.Source
+import scala.util.{ Failure, Success, Try }
+
+import scopt.OParser
+import spray.json._
+import cloudflow.blueprint._
+import cloudflow.blueprint.StreamletDescriptorFormat._
+import cloudflow.blueprint.deployment._
+import cloudflow.extractor._
+
+object Generator {
+
+  val cloudflowVersion = BuildInfo.version
+
+  private def makeCR(appDescriptor: ApplicationDescriptor, version: String): CloudflowCR =
+    CloudflowCR(
+      apiVersion = "cloudflow.lightbend.com/v1alpha1",
+      kind = "CloudflowApplication",
+      metadata = Metadata(
+        annotations = Map("com.lightbend.cloudflow/created-by-cli-version" -> version),
+        labels = Map(
+          "app.kubernetes.io/managed-by" -> "cloudflow",
+          "app.kubernetes.io/part-of" -> appDescriptor.appId,
+          "com.lightbend.cloudflow/app-id" -> appDescriptor.appId),
+        name = appDescriptor.appId),
+      appDescriptor)
+
+  def scanProject(projectId: String, classpath: List[String]): Map[String, com.typesafe.config.Config] = {
```

Comment:
nitpick: maybe ..
```scala
import com.typesafe.config.Config
def scanProject(projectId: String, classpath: List[String]): Map[String, Config] = {
```

because the same config appears later as well

Reply:
fixed